### PR TITLE
hoc2023: AI strings

### PR DESCRIPTION
**Pull request by breville**
This preps the strings we'll need to localise.

### CODE REVIEW

File: apps/i18n/dance/en_us.json [hunk 1]
```diff
@@ -8,6 +8,21 @@
   "aiDanceFeedbackTinkerAiBlock": "Look at the new coding blocks that AI generated. Try choosing a new background or foreground effect.",
   "aiDanceNewMove": "You need to add a block to make the dancer do a new move.",
   "aiDanceStageChangeAfterMeasures": "Make sure to connect the `AI generate stage` block to the `After 4 Measures` event.",
+  "danceAiModalHeading": "generate {input} effect",
```
Comment (breville) [inline apps/i18n/dance/en_us.json:15]: We can probably use a technique like https://github.com/code-dot-org/ml-playground/pull/296 for this.

File: apps/i18n/dance/en_us.json [hunk 2]
```diff
@@ -8,6 +8,21 @@
...
+  "danceAiModalEffectButton": "Effect",
+  "danceAiModalEffect": "A.I. generated this effect.",
+  "danceAiModalUseEffectButton": "Use effect",
+  "danceAiModalCodeButton": "Code",
+  "danceAiModalCode": "Did you know? A.I. wrote this code to generate the effect.",
+  "danceAiModalUseCodeButton": "Use code",
+  "danceAiModalStartOverButton": "Start over",
+  "danceAiModalExplanationButton": "Learn more",
+  "danceAiModalRegenerateButton": "Regenerate",
+  "danceAiModalExplanation": "Each emoji contributed to A.I.'s decision. Here are five possible effects and the calculations that A.I made.",
```
Comment (bencodeorg) [inline apps/i18n/dance/en_us.json]: Period after second A.I here?

**Review by bencodeorg — COMMENTED** [review]
Sorry, I missed your little demo of the new UI flow yesterday so I'm a little out of the loop -- are we removing the bar graphs? I don't see the "background effect" / etc strings from the existing toggle there?

File: apps/i18n/dance/en_us.json (hunk 2 above)
Comment (breville) [reply apps/i18n/dance/en_us.json]: Good catch; fixed!

File: apps/i18n/dance/en_us.json (hunk 1 above)
Comment (fisher-alice) [reply apps/i18n/dance/en_us.json:15]: On ai modal and block, 'effect' is plural ('effects'). Did you intend to change to singular?

Comment (fisher-alice) [reply apps/i18n/dance/en_us.json:15]: Oh! I see it's singular in updated UI (not yet in production).

Comment (breville) [reply apps/i18n/dance/en_us.json:15]: Yes, that's the new block text.

File: apps/i18n/dance/en_us.json [hunk 3]
```diff
@@ -10,8 +10,25 @@
...
+  "danceAiModalEffectButton": "Effect",
+  "danceAiModalEffect": "A.I. generated this effect.",
+  "danceAiModalUseEffectButton": "Use effect",
+  "danceAiModalCodeButton": "Code",
+  "danceAiModalCode": "Did you know? A.I. wrote this code to generate the effect.",
+  "danceAiModalUseCodeButton": "Use code",
+  "danceAiModalStartOverButton": "Start over",
+  "danceAiModalExplanationButton": "Learn more",
+  "danceAiModalRegenerateButton": "Regenerate",
+  "danceAiModalExplanation": "Each emoji contributed to A.I.'s decision. Here are five possible effects and the calculations that A.I. made.",
```
Comment (fisher-alice) [inline apps/i18n/dance/en_us.json]: Just curious. This text is displayed underneath the explanation bar graphs. The sentence  'Here are five possible and the calculations that A.I. made' doesn't seem to match the bar graphs. The bar graphs are showing the calculations for all the possible output effects.

Comment (breville) [reply apps/i18n/dance/en_us.json]: Good observation.  There is a revised design for the explanation mode for which this string will be appropriate.

File: apps/i18n/dance/en_us.json [hunk 4]
```diff
@@ -10,8 +10,25 @@
   "danceAiFeedbackKeyPressRightOrLeft": "⚠️ Make sure to press at least one of the arrow keys while your dance is running.",
   "danceAIFeedbackMatchDancer": "⚠️ You must have the same dancer selected in the make new dancer block and the dancer do new move block.",
```
Comment (fisher-alice) [inline apps/i18n/dance/en_us.json]: Could you make the 'i' lower case in line 11 for 'danceAIFeedbackMatchDancer'? Thanks!

**Review by fisher-alice — APPROVED** [review]

File: apps/i18n/dance/en_us.json (hunk 4 above)
Comment (breville) [reply apps/i18n/dance/en_us.json]: Done.

**Pull request merged**